osidb/models.py

```diff
@@ -1743,6 +1744,16 @@ def _validate_public_source_no_ack(self):
 class FlawCommentManager(ACLMixinManager, TrackingMixinManager):
     """flawcomment manager"""
 
+    def pending(self):
```

Comment:
I am unsure whether we need to handle `pending` comments and whether we should assume that `ATOMIC_REQUESTS` may be False. No action here for you, I am just unsure about it.

Reply:
Good catch, Ondrej already told me to remove mentions of `ATOMIC_REQUESTS`. I'll remove the paragraph.

The method is used in `BugzillaQueryBuilder` so I think we still need it.